src/Simplify.elm

```diff
@@ -5307,7 +5310,20 @@ listAnyChecks checkInfo =
                     ]
 
                 _ ->
-                    []
+                    case Evaluate.isEqualToSomethingFunction checkInfo.firstArg of
+                        Nothing ->
+                            []
+
+                        Just rangesToRemove ->
+                            [ Rule.errorWithFix
+                                { message = "Use List.member instead"
+                                , details = [ "This call to List.any checks for the presence of a value, which what List.member is for." ]
```

Comment:
some words are missing in "[...], which what List.member is for."
If I had a guess, you probably wanted "which is what List.member is for" or "This call [...] checks for [...]. List.member is meant for this exact purpose."